apps/wing-console/console/server/src/utils/license.ts

```diff
@@ -0,0 +1,47 @@
+export const license = `MONADA CONSOLE SOFTWARE
```

Comment:
@skyrpex can explain it better since he proposed this solution, but the loader can not be configured to load md files in the UI package.


This is the error I was getting:
```
ESM Build failed
Error: Build failed with 1 error:
src/layout/vscode-layout.tsx:12:20: ERROR: No loader is configured for ".md" files: ../../LICENSE.md?raw
```

Reply:
I proposed to have the license text in a variable because it seemed the most simple way. The bundler in the console server (tsup) is not configured to import files other than ts/js.

I've been looking around and we could configure tsup with an esbuild raw loader such as https://github.com/hannoeru/esbuild-plugin-raw, then we could import the md file like this:

```ts
import license from './license.md?raw';
```

Reply:
https://github.com/winglang/wing/pull/3338/commits/7e1f042595b723b7981f62fb144210c4cfe12a10

Reply:
Looks good!